data/config-files/server.conf

```diff
@@ -0,0 +1,16 @@
+admin_group: root
+database_config:
+        host: 127.0.0.1
+        port: 1
+        username: postgres
+        password: test123
+email_config:
+        host: 127.0.0.1
+        port: 1
+        username: ammen99
+        password: ammen99
+special_resources:
+        test: 10
+blocking_enabled: True
```

Comment:
Do we still need this attribute?

Reply:
yes, because it is required by the ServerConfig class here: https://github.com/DistributedTaskScheduling/JobAdder/blob/master/src/ja/server/config.py#L141